trace/backend_collector.go

```diff
@@ -252,11 +247,27 @@ func (c *BackendCollector) uploadWitnesses(in []interface{}) {
 	}
 }
 
+func (c *BackendCollector) periodicFlush() {
```

Comment:
Where does this get called?

Reply:
Good catch, I had intended to call it in NewBackendCollector as a goroutine.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/17/commits/f88f0281d685bc49f4daa35458ccd61036862fde